client/src/components/UIElements/Button.scss

```diff
@@ -0,0 +1,14 @@
+.button {
+    font-size: 16px;
+    min-width: 100px;
+    font-weight: 600;
+    height: 40px;
+    padding: 5px;
+    margin: 5px;
+    color: white;
+    background-color: #2a2a67;
```

Comment:
I have an idea.
We can create a global.scss with variables like colors.
An than we can even add a light/dark mode.
We can talk about it if you want.